sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/OpenAISyncClientTest.java

```diff
@@ -402,4 +407,110 @@ public void testChatCompletionsStreamingBasicSearchExtension(HttpClient httpClie
+
+        getAudioTranslationRunner((deploymentName, fileName) -> {
+            byte[] file = BinaryData.fromFile(openTestResourceFile(fileName)).toBytes();
+            AudioTranslationOptions translationOptions = new AudioTranslationOptions(file);
+            translationOptions.setResponseFormat(AudioTranscriptionFormat.TEXT);
+
+            String transcription = client.getAudioTranslationText(deploymentName, translationOptions, fileName);
+            assertEquals("It's raining today.\n", transcription);
+        });
+    }
+
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.openai.TestUtils#getTestParameters")
+    public void testGetAudioTranslationTextPlainWrongFormats(HttpClient httpClient, OpenAIServiceVersion serviceVersion) {
+        client = getOpenAIClient(httpClient, serviceVersion);
+
+        getAudioTranslationRunner((deploymentName, fileName) -> {
+            byte[] file = BinaryData.fromFile(openTestResourceFile(fileName)).toBytes();
+            AudioTranslationOptions translationOptions = new AudioTranslationOptions(file);
+
+            translationOptions.setResponseFormat(AudioTranscriptionFormat.JSON);
+            assertThrows(IllegalArgumentException.class, () -> {
+                client.getAudioTranslationText(deploymentName, translationOptions, fileName);
+            });
+
+            translationOptions.setResponseFormat(AudioTranscriptionFormat.VERBOSE_JSON);
+            assertThrows(IllegalArgumentException.class, () -> {
+                client.getAudioTranslationText(deploymentName, translationOptions, fileName);
+            });
+        });
+    }
+
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.openai.TestUtils#getTestParameters")
+    public void testGetAudioTranslationJsonWrongFormats(HttpClient httpClient, OpenAIServiceVersion serviceVersion) {
+        client = getOpenAIClient(httpClient, serviceVersion);
+
+        getAudioTranslationRunner((deploymentName, fileName) -> {
+            byte[] file = BinaryData.fromFile(openTestResourceFile(fileName)).toBytes();
+            AudioTranslationOptions translationOptions = new AudioTranslationOptions(file);
+
+            translationOptions.setResponseFormat(AudioTranscriptionFormat.TEXT);
+            assertThrows(IllegalArgumentException.class, () -> {
+                client.getAudioTranslation(deploymentName, translationOptions, fileName);
+            });
+
+            translationOptions.setResponseFormat(AudioTranscriptionFormat.SRT);
+            assertThrows(IllegalArgumentException.class, () -> {
+                client.getAudioTranslation(deploymentName, translationOptions, fileName);
+            });
+
+            translationOptions.setResponseFormat(AudioTranscriptionFormat.VTT);
+            assertThrows(IllegalArgumentException.class, () -> {
+                client.getAudioTranslation(deploymentName, translationOptions, fileName);
+            });
```

Comment:
NIT: For code clean, we might able to not repeat
```
TranscriptionFormat transcriptionFormat = ..'
translationOptions.setResponseFormat(transcriptionFormat.);
assertThrows(IllegalArgumentException.class, () -> {
    client.getAudioTranslation(deploymentName, translationOptions, fileName);
});

Reply:
Not entirely sure how I would be able to achive that. What would you suggest?

Reply:
Suggestion provided in https://github.com/Azure/azure-sdk-for-java/pull/36693#discussion_r1326183120